examples/testing-module/Makefile

```diff
@@ -111,12 +111,16 @@ check-mtf-options:
 	cd $(TEMPDIR); grep 'MODULE=docker' log
 	rm -rf "$(TEMPDIR)"
 
+MODNVR = testmodule-master-20180112204908
+COMMITHASH = e3e099a969c2f88204ae4661924e3930f017a6bb
```

Comment:
This will cause troubles, once the commithash is changed. Is there any possibility, how to do it dynamically?

Reply:
This cannot change.